garage/misc/logger/logger_utils.py

```diff
@@ -0,0 +1,58 @@
+# flake8: noqa
+from enum import Enum
+import json
+import os
+
+from garage.misc.autoargs import get_all_parameters
+from garage.misc.console import mkdir_p
+
+
+def log_parameters_lite(log_file, args):
```

Comment:
Not sure what this even does. Do you want me to get rid of it?

Reply:
OK. I didn't want to mess with this code because it was copied from the old logger, but I'll delete `log_parameters` and renames `log_parameters_lite` to `log_parameters`